x-pack/test/plugin_functional/config.ts

```diff
@@ -28,7 +28,8 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     // list paths to the files that contain your plugins tests
     testFiles: [
       resolve(__dirname, './test_suites/resolver'),
-      resolve(__dirname, './test_suites/global_search'),
+      // Disabling this suite as it is broken.
+      // resolve(__dirname, './test_suites/global_search'),
```

Comment:
@pgayvallet @myasonik @ryankeairns @cchaos This PR would disable the `global_search` suite (it is currently broken) and enable the entire `x-pack/test/plugin_functional` config. Is that OK with y'all? Thank you

Reply:
```
GlobalSearch API
       GlobalSearchBar
         basically works:
Error: expected 0 to equal 15
```

Seems only this test is failing, could you add a skip on the root describe in `x-pack/test/plugin_functional/test_suites/global_search/global_search_bar.ts` instead?

Reply:
And add a comment referencing https://github.com/elastic/kibana/issues/81397. Thank you

Reply:
Will do

Reply:
Seems you got other failures from the suite. Forget that, just skip the whole suite.

Reply:
ty, will do.